.github/CONTRIBUTING.md

```diff
@@ -210,7 +210,7 @@ var/path_type = "/obj/item/baseball_bat"
 While DM allows other ways of declaring variables, this one should be used for consistency.
 
 ### Tabs, not spaces
-You must use tabs to indent your code, NOT SPACES.
+You should use tabs to indent your code, NOT SPACES.
```

Comment:
Mixing tabs and spaces requires immediate banishment to the depths of hell, do not encourage this heathen behaviour

Reply:
this is also linted against